Author:	Ulf Samuelsson
Date:	2007-01-25

Log:
Patch adding "sys/acl.h" needed to compile mtd.

Modified:
	package/mtd/mtd_acl.patch

Changeset
	package/mtd/mtd_acl.patch

diff -urN mtd_orig-0rig/include/sys/acl.h mtd_orig/include/sys/acl.h
--- mtd_orig-0rig/include/sys/acl.h	1970-01-01 01:00:00.000000000 +0100
+++ mtd_orig/include/sys/acl.h	2007-01-17 20:11:22.000000000 +0100
@@ -0,0 +1,32 @@
+/*
+  File: linux/posix_acl.h
+
+  (C) 2002 Andreas Gruenbacher, <a.gruenbacher@computer.org>
+*/
+
+
+#ifndef __LINUX_POSIX_ACL_H
+#define __LINUX_POSIX_ACL_H
+
+#define ACL_UNDEFINED_ID	(-1)
+
+/* a_type field in acl_user_posix_entry_t */
+#define ACL_TYPE_ACCESS		(0x8000)
+#define ACL_TYPE_DEFAULT	(0x4000)
+
+/* e_tag entry in struct posix_acl_entry */
+#define ACL_USER_OBJ		(0x01)
+#define ACL_USER		(0x02)
+#define ACL_GROUP_OBJ		(0x04)
+#define ACL_GROUP		(0x08)
+#define ACL_MASK		(0x10)
+#define ACL_OTHER		(0x20)
+
+/* permissions in the e_perm field */
+#define ACL_READ		(0x04)
+#define ACL_WRITE		(0x02)
+#define ACL_EXECUTE		(0x01)
+//#define ACL_ADD		(0x08)
+//#define ACL_DELETE		(0x10)
+
+#endif  /* __LINUX_POSIX_ACL_H */
